.github/workflows/test.yml

```diff
@@ -626,3 +626,29 @@ jobs:
       - name: Test ${{ matrix.pkg }} Build
         run: |
           nix develop -c sh -c "cd pkg/${{ matrix.pkg }} ; zig build test"
+
+  test-debian-12:
```

Comment:
Oh also, this key needs to be added to the "checks required" job at the top of this file.

Reply:
Fixed.